src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageException.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace System.IO.IsolatedStorage
+{
+    [System.Runtime.InteropServices.ComVisible(true)]
+    public class IsolatedStorageException : Exception
+    {
+        private const int COR_E_ISOSTORE = unchecked((int)0x80131450);
+
+        internal Exception m_UnderlyingException;
```

Comment:
It looks like this exists purely for the debugger to access?  It'd be worth adding a comment to that effect.